flocker/node/functional/test_deploy.py

```diff
@@ -51,13 +53,16 @@ def __init__(self, hostname, volume_service, docker_client=None,
         self.docker_client = self.applications_deployer.docker_client
         self.network = self.applications_deployer.network
 
-    def discover_state(self, cluster_state):
-        d = self.manifestations_deployer.discover_state(cluster_state)
+    def discover_state(self, cluster_state, persistent_state):
+        d = self.manifestations_deployer.discover_state(
+            cluster_state, persistent_state=persistent_state)
 
         def got_manifestations_state(manifestations_local_state):
             manifestations_state = manifestations_local_state.node_state
             app_discovery = self.applications_deployer.discover_state(
-                DeploymentState(nodes={manifestations_state}))
+                DeploymentState(nodes={manifestations_state}),
+                persistent_state=PersistentState(),
```

Comment:
:grey_question: Should this really be an empty persistent state, or should we instead use a constant in this file? Concerned because if we are going to add fields to PersistentState, I would rather minimize the number of times I had to go around and add a default value for those fields throughout the tests.